src/utils/source-maps.js

```diff
@@ -37,9 +40,13 @@ export async function getMappedLocation(
   state: Object,
   sourceMaps: Object,
   location: Location
-) {
+): Promise<Location> {
   const source = getSource(state, location.sourceId);
 
+  if (!source) {
+    throw new Error("Unknown source for location");
```

Comment:
@ravi610 Actually, looking at this again it _might_ be more in line with the old behavior to do `return location;` here. Would you be up for making a new PR with that tweak?

Reply:
Done in https://github.com/devtools-html/debugger.html/pull/7176